app/index.html

```diff
@@ -128,6 +128,25 @@ <h2 class="title-margin-bottom">News &#x26; Events</h2>
                
                <div class="eight columns">
                    
+                   <h4>Third Annual Cornell Fencing Association Fall Dinner</h4>
+                   <img class="news-img" src="images/Cornell20180915Fencing55.jpg" alt="Alumni and spouses at the dinner" />
+                   <img class="news-img" src="images/Cornell20180915Fencing47.jpg" alt="Alumni and spouses at the dinner" />                
+                   <img class="news-img" src="images/Cornell20180915Fencing.jpg" alt="Coaches and Guests" />
+
+                   <p>September 15, 2018 at the New York Athletic Club</p>
+
+                   <p>Alumni, Olympians, World Team members and Olympic Coaches wined, dined and some even fenced the night away at our 3rd Annual CFA Fall Dinner. In what portends to be a winning season ahead, Head Coach Daria Schneider, the evening&#x2019;s MC,  introduced us to Cornell&#x2019;s newest coach, Omar Elgeziry. A 2016 Olympian himself, he was previously assistant coach at Air Force for 3 seasons and coached 11 athletes into qualifying positions in the NCAA. He also spent one year as the national epee coach for the USA Pentathlon. We welcome him to the Cornell family.</p>
+                   
+                   <p>We also said our farewell to Nicole Ross, who is taking a leave of absence to prepare for qualification to the 2020 Olympic Team.  As current world champion, we know she will continue to build on her success. She spoke about her enjoyment with working with the team and being a member of the Cornell fencing family. We wish her all the best! </p>

```

Comment:
Let's try to be consistent on spaces after the period. The previous paragraph had a 1-spacer, and this paragraph had a 2-spacer.

Reply:
This is likely due to me direct copying text from Nina. Luckily HTML will kindly mash all extra spaces down to one so only code viewers will notice it :)